data/human/news.txt

```diff
@@ -1990,6 +2093,75 @@ news "conspiracy theorist"
 
 
 
+news "players crew member"
+	location
+		government "Republic" "Free Worlds" "Neutral" "Independent" "Syndicate"
+	to show
+		"flagship crew" > 30
+	name
+		word
+			"Some crew member"
+	message
+		word
+			`"`
+		word
+			"I'm serving onboard the '<ship>', but I've never met the captain. Can you imagine that?"
+			"Hey captain, good to see you here. I'm looking forward to the next trip (and I'm looking forward to my 100 credits a day)."
+			"You look like a poor excuse for a captain. I'm serving onboard the '<ship>' and I'm sure that my captain is far better dressed than you."
+		word
+			`"`
+
+
+
+news "sticker on ship"
+	location
+		government "Republic" "Free Worlds" "Neutral" "Independent" "Syndicate"
+	name
+		word
+			"Sticker on ship"
+	message
+		word
+			"I brake for asteroid snails."
+			"Student pilot."
+			"Babe onboard."
```

Comment:
Is this supposed to be 'baby' or..?

Reply:
Sure, that is also nice to put on a sticker.